test/fixtures/confidential_instance_template/network.tf

```diff
@@ -0,0 +1 @@
+../shared/network.tf
```

Comment:
copyright?  this file seems odd

Reply:
It is a symlink, not the clearest in the UI.